core/src/components/modal/modal.tsx

```diff
@@ -820,7 +880,16 @@ export class Modal implements ComponentInterface, OverlayInterface {
         {mode === 'ios' && <div class="modal-shadow"></div>}
 
         <div role="dialog" class="modal-wrapper ion-overlay-wrapper" part="content" ref={(el) => (this.wrapperEl = el)}>
-          {showHandle && <div class="modal-handle" part="handle"></div>}
+          {showHandle && (
+            <button
+              class="modal-handle"
+              // Prevents the handle from receiving keyboard focus when it does not cycle
+              tabIndex={!isHandleCycle ? -1 : 0}
+              aria-label={isHandleCycle ? 'Activate to adjust the size of the dialog overlaying the screen' : null}
+              onClick={this.onHandleClick}
```

Comment:
We should remove the event listener if `!isHandleCycle`. I believe some screen readers will indicate that clicking the handle is supposed to do something if there's a click listener.